state_machine/modules/json_conf/include/klepsydra/state_machine/json_to_state_machine.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 Klepsydra Technologies AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cereal/types/string.hpp>
+#include <cereal/types/vector.hpp>
```

Comment:
what about json.hpp?

Reply:
Not required at this point (how to serialize), included at `json_config_loader.cpp` (built and tested).

Reply:
1. Fresh build and installation of `kpsr-core`
2. kpsr-tutorial builds and tests ok, see PR #KCE-65-JSON-at-SM-tutorial

The minimal headers' set to install is just `json_config_loader.h` to run `example1a.cpp`.
Note the source and install folders are `state_machine/`, not `conf/`.